src/Components/WebAssembly/Build/test/BuildIntegrationTests/PublishIntegrationTest.cs

```diff
@@ -101,6 +101,38 @@ public async Task Publish_InRelease_Works()
             Assert.FileCountEquals(result, 1, publishDirectory, "*", SearchOption.TopDirectoryOnly);
         }
 
+        [Fact]
+        public async Task Publish_WithExistingWebConfig_Works()
+        {
+            // Arrange
+            using var project = ProjectDirectory.Create("standalone", additionalProjects: new[] { "razorclasslibrary", "LinkBaseToWebRoot" });
+            project.Configuration = "Release";
+
+            var webConfigContents = @"<?xml version='1.0' encoding='utf-8' ?>
+< configuration >
+  < system.web >
+  </ system.web >
+</ configuration >";
```

Comment:
Looks like VS reformatted this for you :)

I would suggest either:

 * If this is meant to be a valid `web.config` file, remove the extra spaces
 * Or if it doesn't have to be valid, replace the whole thing with something like `var webConfigContents = "test web.config contents";`